src/dlgPackageExporter.cpp

```diff
@@ -382,7 +382,7 @@ bool dlgPackageExporter::eventFilter(QObject* obj, QEvent* evt)
             for (int i = mDescriptionImages.size() - 1; i >= 0; i--) {
                 QString fname = mDescriptionImages.at(i);
                 QFileInfo info(fname);
-                fname.replace(" ", "%20");
+                fname = fname.toHtmlEscaped();
```

Comment:
toHtmlEscaped() unfortunately doesn't seem to convert " " (space) to "%20" which means filenames with a space in it won't work any more with this change

Reply:
Ah ok.